source/app/models/RockPaperScissor.rb

```diff
@@ -0,0 +1,49 @@
+class RockPaperScissors
+  attr_reader :player1_id
+  def initialize(player1_id, player2_id, token1, token2)
+    @player1 = player1_id
+    @player2 = player2_id
+    @token1 = token1
+    @token2 = token2
+  end
+
+  def play
+    token1, token2 = @token1, @token2
+    player1, player2 = @player1, @player2
+    if token1 == token2
+      draw
+    end
+    case token1
```

Comment:
Okay, here is where things get a little bit to "complex" we really only have 3 cases right? 

{ scissor: paper,
  paper: rock,
  rock:  scissor } 
 
then we can just give it player_1's token .... and if it does not return player 2's token as the value, we know we lost... 

if the two tokens are the same it's a tie!